uitools/import/Esri/ArcGISRuntime/Toolkit/BasemapGallery.qml

```diff
@@ -228,16 +239,21 @@ Pane {
             ToolTip.text: modelData.tooltip
             MouseArea {
                 id: mouseArea
+                z : 2
```

Comment:
❓ What does this do?

Reply:
I set the mouse area above everything else, so when a click happens it is recorded by the area and not from the Image. Otherwise only the bottom part of the item is clickable

Reply:
Does this still happen with the change to `ItemDelegate`?

Reply:
Yes, but is because I have to set the [`z`](https://github.com/Esri/arcgis-runtime-toolkit-qt/pull/462/files#diff-ca77a4aefb9091361317a2c42ed3a2b69aaafbb1a6ae456189aec5cf38814285R202) property of the indicator object > 0, otherwise is rendered behind the `Image`, but doing so, the indicator will be over the `MouseArea`

https://user-images.githubusercontent.com/91129623/142215849-c785ca96-279a-4301-8824-a331ae83b573.mp4

.

Reply:
I see, that makes sense. Good find and good fix!